src/game/overlay.c

```diff
@@ -78,14 +79,105 @@ static RGB888 m_ColorBarMap[][COLOR_STEPS] = {
       { 120, 60, 70 } },
 };
 
-static void Overlay_GetBarLocation(
-    int8_t bar_location, int32_t width, int32_t height, int32_t *x, int32_t *y);
-static void Overlay_DrawBar(int32_t value, int32_t value_max, int32_t bar_type);
+typedef struct BAR_INFO {
+    BAR_TYPE type;
+    int32_t value;
+    int32_t max_value;
+    int32_t percent;
+    bool show;
+    T1M_BAR_SHOW_MODE show_mode;
+    bool blink;
+    int32_t blink_counter;
+    int32_t timer;
+    int32_t color;
+    T1M_BAR_LOCATION bar_location;
```

Comment:
IMO we can drop `T1M_` prefix from these enums, and simplify `bar_location` to `location`